hack/deploy_cincinnati.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# The script serves as an example of how Cincinnati can be run on an OpenShift cluster. 
+
+# Create a dummy secret as a workaround to not having real secrets
+oc create secret generic cincinnati-credentials --from-literal=""
```

Comment:
This needs to be moved after `oc new-project cincinnati`

Reply:
Done by f0b1bd460325fa2d11b1cdca9574a278cf66580e.